### DISCUSSION TITLE
Use Radios instead of Checkboxes for single select lists

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7761] [Feature Requests]
In the Directus App, when we're put into a context (eg: Layouts, or drawer) where we can only select **one** item, it would be ideal if the UI showed radio buttons instead of checkboxes.

https://fonts.google.com/icons?selected=Material+Icons+Outlined:dynamic_feed&icon.query=radio+button

**Comment by nickrum** [comment]
This has since been addressed 👍
